src/xcode/ENA/ENA/Source/Services/Risk/__tests__/HourKeyPackagesDownloadTests.swift

```diff
@@ -293,7 +293,7 @@ class HourKeyPackagesDownloadTests: XCTestCase {
 	func test_When_PersistHourPackagesToDatabaseFails_Then_unableToWriteDiagnosisKeysErrorReturned() {
 		let store = MockTestStore()
 
-		let packagesStore = DownloadedPackagesStoreErrorStub(error: .unknown)
+		let packagesStore = DownloadedPackagesStoreErrorStub(error: DownloadedPackagesSQLLiteStore.StoreError.sqliteError(.unknown))
```

Comment:
(not blocking) maybe we can declare the `DownloadedPackagesStoreErrorStub ` error type as `StoreError` to simplify the code again or was there a reason to go fully verbose here?

Reply:
Lets tackle this with the next change.

Reply:
Just mentioning. I'm not insisting on this one yet.